tests/passing-configs/index.js

```diff
@@ -14,8 +15,6 @@ export function filePathToConfigObj(filepath) {
   return {
     config,
     filepath,
-    name: path.parse(filepath).name,
```

Comment:
Just saw @sarbbottam's note. So I've updated the command to allow you to also edit the commit message at the same time :-)